libs/model/src/middleware/auth.ts

```diff
@@ -288,6 +288,11 @@ async function checkGatedActions(
   action: GroupGatedActionKey,
   topic_id: number,
 ): Promise<void> {
+  // AI bot user bypasses all gating restrictions
+  if (actor.user.id && (await isBotUser(actor.user.id))) {
```

Comment:
is this a light operation? if not, we should probably pass an argument to only check when necessary